Longitude should range from -180 to 180 not -90 to 90

Hi,

```
2024-11-15 08:57:03,969 - ERROR [kytos.napps.kytos/sdx] [main.py:388:post_topology_to_sdxlc] (AnyIO worker thread) Failed to send topoloty to SDX-LC: {
  "detail": "-110.0 is less than the minimum of -90.0 - 'nodes.12.location.longitude'",
  "status": 400,
  "title": "Bad Request",
  "type": "about:blank"
}
 - Traceback: Traceback (most recent call last):,   File "//var/lib/kytos/napps/kytos/sdx/main.py", line 384, in post_topology_to_sdxlc,     assert response.status_code == 200, response.text, AssertionError: {,   "detail": "-110.0 is less than the minimum of -90.0 - 'nodes.12.location.longitude'",,   "status": 400,,   "title": "Bad Request",,   "type": "about:blank", }, ,
```

It is easy to find specs on the web which clarify the allowed range for longitude as being from -180 to 180. One example (https://developers.google.com/maps/documentation/javascript/reference/coordinates#LatLng)

> Latitude ranges between -90 and 90 degrees, inclusive. Values above or below this range will be clamped to the range [-90, 90]. This means that if the value specified is less than -90, it will be set to -90. And if the value is greater than 90, it will be set to 90.
> Longitude ranges between -180 and 180 degrees, inclusive. Values above or below this range will be wrapped so that they fall within the range. For example, a value of -190 will be converted to 170. A value of 190 will be converted to -170. This reflects the fact that longitudes wrap around the globe.


The topology data model seems wrong on this point. It will also have to be fixed.

--- a/sdx_lc/swagger/swagger.yaml
+++ b/sdx_lc/swagger/swagger.yaml
@@ -1183,8 +1183,8 @@ components:
           maximum: 90.0
         longitude:
           type: number
-          minimum: -90.0
-          maximum: 90.0
+          minimum: -180.0
+          maximum: 180.0
         iso3166_2_lvl4:
           type: string
           minLength: 5

